os/include/time.h

```diff
@@ -245,6 +245,16 @@ int clock_gettime(clockid_t clockid, FAR struct timespec *tp);
  */
 int clock_getres(clockid_t clockid, FAR struct timespec *res);
 
+/**
+ * @ingroup TIME_KERNEL
+ * @brief returns the implementation's best approximation to thei
+ * processor time used by the process since the beginning of a
+ * implementation-defined era related only to the process invocation.
+ * @details @b #include <time.h> \n
+ * POSIX API (refer to : http://pubs.opengroup.org/onlinepubs/9699919799/)
+ * @since TizenRT v1.1
```

Comment:
1. This should be 2.0.
2. To leave this comment, (This comment means this API is tested and passed and will be shown at API reference)
   we must make utc in testcase folder. But there is no tc for this.
  Will you make a tc before applying test by SQE? It will be next week.